ExampleMod/Common/GlobalProjectiles/ExampleGlobalProjectile.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ExampleMod.Content.NPCs;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.GlobalProjectiles
+{
+	// This file shows an example of a GlobalProjectile class. GlobalProjectile hooks are called on all projectiles in the game and are suitable for sweeping
+	// changes like adding additional data to all projectiles in the game. Here we simply override PostAI in order to make Purification Power loop through the
+	// npc array to affect Wraiths specifically, transforming them into Fallen Souls, as it is simple to understand.
+	public class ExampleGlobalProjectile : GlobalProjectile
```

Comment:
this should probably be renamed to `WraithPurificationGlobalProjectile` and put in the same folder as the `ExampleFallenSoul` NPC.

I'm not sure what we decided with ExampleMod, but we wanted more self-contained content examples in their own subfolders. If we just want to showcase how to use GlobalProjectile, we should use a more standalone example.